models/netsuite/stg_netsuite__accounts.sql

```diff
@@ -41,7 +41,7 @@ final as (
         _fivetran_deleted
 
         --The below macro adds the fields defined within your accounts_pass_through_columns variable into the staging model
-        {{ fivetran_utils.fill_pass_through_columns('accounts_pass_through_columns') }}
+        {{ netsuite_source.fill_pass_through_columns(var('accounts_pass_through_columns')) }}
```

Comment:
Shouldn't we also add a default value for these just in case?

Reply:
Added a default [] value to each of the new macro calls.